### PR TITLE
Begin README.md and add images.

### PR DESCRIPTION
**Pull request by scottdangelo**
Initial README.md. Not complete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,166 @@
...
+
+* [IBM Cloud Pak for Data](https://www.ibm.com/analytics/cloud-pak-for-data)
+* [Watson Machine Learning Add On for ICP4D](https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/com.ibm.icpdata.doc/zen/admin/add-ons.html#add-ons__ai)
+
+# Steps
+
+1. [Create a new Project](#1-create-a-new-project)
+1. [Upload the dataset](#2-upload-the-dataset) if you are not on the [ICP4D Learning Path]().
+1. [Import notebook to Cloud Pak for Data](#3-import-notebook-to-cloud-pak-for-data)
+1. [Import dataset into the notebook](#4-import-dataset-into-the-notebook)
```
Comment (rhagarty) [inline README.md:46]: link doesn't work

File: README.md [hunk 2]
```diff
@@ -1,2 +1,166 @@
...
+
+![Add notebook name and URL](doc/source/images/wml-2-add-name-and-url.png)
+
+When the Jupyter notebook is loaded and the kernel is ready then we can start executing cells.
+
+![Notebook loaded](doc/source/images/wml-3-notebook-loaded.png)
+
+### 4.0 Import dataset into the notebook
+
+Spend an minute looking through the sections of the notebook to get an overview.
```
Comment (rhagarty) [inline README.md]: spend "a" minute

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md:46]: fixed

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: fixed.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**